lms/djangoapps/verify_student/views.py

```diff
@@ -1047,3 +1049,82 @@ def reverification_window_expired(_request):
     """
     # TODO need someone to review the copy for this template
     return render_to_response("verify_student/reverification_window_expired.html")
+
+
+class ICRVReverifyView(View):
```

Comment:
"ICRV" is a little hard to understand.  I'd prefer the longer name: `InCourseReverifyView`